modules/rosa-edit-objects.adoc

```diff
@@ -197,11 +197,26 @@ Set 4 replicas on a machine pool named `mp1` on a cluster named `mycluster`.
 
 [source,terminal]
 ----
-$ rosa edit machinepool --cluster=mycluster mp1 --replicas=4
+$ rosa edit machinepool --cluster=mycluster --replicas=4 --name=mp1
 ----
 
 Enable autoscaling on a machine pool named `mp1` on a cluster named `mycluster`.
 
 [source,terminal]
 ----
-$ rosa edit machinepool --cluster-name=mycluster --name=mp1 --enable-autoscaling --min-replicas=3 --max-replicas=5
+$ rosa edit machinepool --cluster-name=mycluster --enable-autoscaling --min-replicas=3 --max-replicas=5 --name=mp1
+----
+
+Disable autoscaling on a machine pool named `mp1` on a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa edit machinepool --cluster-name=mycluster  --enable-autoscaling=false --replicas=3 --name=mp1
+----
+
+Reshape the autoscaling range on a machine pool named `mp1` on a cluster named `mycluster`.
```

Comment:
```suggestion
Modify the autoscaling range on a machine pool named `mp1` on a cluster named `mycluster`.
```

Can you both increase and decrease the range or just increase the range? If you can only increase the range, s/Reshape/Increase.

Reply:
It can be increased or decreased. Thanks for the clarification of termonology. I have changed it.